src/main/kotlin/no/nav/klage/dokument/service/distribusjon/DokumentEnhetDistribusjonService.kt

```diff
@@ -23,12 +23,18 @@ class DokumentEnhetDistribusjonService(
     fun distribuerDokumentEnhet(dokumentEnhet: DokumentEnhet): DokumentEnhet =
         if (!dokumentEnhet.erAvsluttet()) {
             logger.debug("dokumentEnhet ${dokumentEnhet.id} er ikke distribuert")
```

Comment:
Merker jeg blir litt forvirret når `avsluttet` betyr samme som `distribuert`. Kan godt være de er synonymer i `kabal-document`. Nevner det i tilfelle det kommer frem bedre forslag for hva vi kaller ting. I mitt hode kan ting være (i det minste delvis) distribuert uten at hele dokumentEnheten er avsluttet.

Reply:
Går litt på hva vi logger her.

Reply:
Ja, ting kan være (delvis) distribuert, så det blir avsluttet først når det er helt distribuert. Sånn sett blir avsluttet og helt distribuert synonymer.

Reply:
Jeg endret den ene log-statementen